### ISSUE TITLE
Allow for formatted raw HTML to be included in DOCX via "altChunk"

### ISSUE THREAD

**Issue by mjfs** [jgm/pandoc#3883]
Open XML (.DOCX) supports "**altChunk**" markup ("w:altChunk" element) that enables inclusion of raw HTML document into Open XML compliant document. By supporting raw HTML in Pandoc it would be possible to output HTML segments used in markdown documents unmodified to DOCX output.

More details on the DOCX's altChunk functionality:
-   <https://blogs.msdn.microsoft.com/ericwhite/2008/10/26/how-to-use-altchunk-for-document-assembly/>

Raw HTML output would me especially useful for pre-processors of markdown files. In this way a HTML formatted table that is an output of a code segment compatible with pre-processor would still keep its format when the chosen target file format would be DOCX.

A concrete use case is R's output when used in combination with Knitr and xtable/stargazer packages:
-   <https://stackoverflow.com/questions/18961681/how-to-display-beautiful-glm-and-multinom-table-with-rmd-and-knit-html#18968292>

**Comment by jgm** [comment]
I didn't know about this; it seems like a great idea.
It should be made sensitive to the `raw_html` flag, so it
can be turned off (e.g. when someone wants to provide
both a raw HTML and a raw openxml version of something,
for different output formats).

**Comment by jgm** [comment]
It would be helpful to have a sample of the actual XML required, not just code in the Microsoft SDK.
Or a docx with an altChunk.

**Comment by mjfs** [comment]
I have updated the content above with an example. For a on-the-fly example one can use a [html-docx-js](https://github.com/evidenceprime/html-docx-js) library that performs the conversion from HTML to DOCX via altChunk in the browser. There is an on-line example available in project's [readme](http://evidenceprime.github.io/html-docx-js/test/sample.html).

Also a single XML file instead of an archive that is the standard use case would also be possible with DOCX. It is therefore also possible to convert between one and the other:
-    [Transforming Open XML Documents to Flat OPC Format](https://blogs.msdn.microsoft.com/ericwhite/2008/09/29/transforming-open-xml-documents-to-flat-opc-format/)

**Comment by tballison** [comment]
> I have updated the content above with an example.
@mjfs , would you be willing to contribute the example files under Apache Software License 2.0 so that we can use them in our unit tests on Apache Tika (https://issues.apache.org/jira/browse/TIKA-4124)?  Thank you!

**Comment by mjfs** [comment]
@tballison, of course. Consider examples released under Apache Software License 2.0.
